frontends/web/src/components/icon/icon.tsx

```diff
@@ -52,6 +52,7 @@ import saveSVG from './assets/icons/save.svg';
 import saveLightSVG from './assets/icons/save-light.svg';
 import starSVG from './assets/icons/star.svg';
 import starInactiveSVG from './assets/icons/star-inactive.svg';
+import selectedCheckLight from './assets/icons/selected-check-light.svg';
```

Comment:
nit: for consistency add SVG at the end as all other imported svgs end with SVG in the imoported name.

Reply:
oops..